File: 3195-separate-black-and-white-balls/3195-separate-black-and-white-balls.java
class Solution {
    public long minimumSteps(String s) {
        long min_swap = 0;

        int l = 0;
        for(int r = 0; r < s.length(); ++r) {
            if(s.charAt(r) == '0') {
                min_swap += (r - l);
                l++;
            }
        }

        return min_swap;
    }
}

File: 3195-separate-black-and-white-balls/3195-separate-black-and-white-balls.py
class Solution:
    def minimumSteps(self, s: str) -> int:
        min_swap = 0

        l = 0

        for r in range(len(s)):
            if s[r] == "0":
                min_swap += (r - l)
                l += 1
        

        return min_swap
        